pkg/apis/configuration/validation/validation.go

```diff
@@ -95,6 +95,23 @@ func validateTime(time string, fieldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+// http://nginx.org/en/docs/syntax.html
+var validNginxSize = regexp.MustCompile(`^\d+[kKmMgG]?$`)
```

Comment:
In `validation.go` when we use regular expressions, we follow a convention. Please take a look at this example https://github.com/nginxinc/kubernetes-ingress/blob/65e08c37ed04425855a562f22b966dd07d257abe/pkg/apis/configuration/validation/validation.go#L526 and this example https://github.com/nginxinc/kubernetes-ingress/blob/65e08c37ed04425855a562f22b966dd07d257abe/pkg/apis/configuration/validation/validation.go#L276
Could you possibly follow that convention?